microsoft-edge/devtools-guide-chromium/javascript/reference.md

```diff
@@ -366,6 +366,20 @@ If you find yourself running the same debug code in the **Console** tool over an
 See [Run snippets of JavaScript on any webpage](./snippets.md).
 
 
+<!-- ====================================================================== -->
+## Use `//# sourceURL` to name evaluated files in the Sources tool
+
+The `//# sourceURL` pragma, such as `//# sourceURL=file-name.js`, is a special comment that makes it easier to work with evaluated JavaScript code and JavaScript code found in `<script>` tags in an HTML document.
+
+When JavaScript code is run by the browser, DevTools can only name it if it was loaded from a file. When running JavaScript directly from a `<script>` tag or by using the `eval()` function, the **Sources** tool doesn't have a file name to display these files in the **Navigator** pane.
+
+Use the `//# sourceURL` pragma to give a name to evaluated files, inline scripts, and styles, so that each one appears as a recognizable file name in the **Sources** tool.  For example:
```

Comment:
pls use consistent order of the items throughout the doc/section:
* evald files
* inline <script> blocks
* styles -- be more technically specific? eg:   style rules; CSS style rules (or whatever tech term).  Pls be technically more specific than "styles".